Sources/Overload/OvTools/src/OvTools/Utils/String.cpp

```diff
@@ -78,3 +78,27 @@ std::string OvTools::Utils::String::GenerateUnique(const std::string& p_source,
 
     return result;
 }
+
+void OvTools::Utils::String::Trim(std::string& p_str, const ETrimType p_trimType)  
+{  
+   if (p_trimType & ETrimType::Leading || p_trimType & ETrimType::TwoWays)  
```

Comment:
It seems like your enum tries to behave like flags. Therefore, `ETrimType::TwoWays` should be set to `3`, or `ETrimType::Leading | ETrimType::Trailing`.

This check would only need to be:
```cpp
if (p_trimType & ETrimType::Leading) {}
```